packages/did-core-test-server/report/report-template.html

```diff
@@ -301,6 +301,12 @@ <h1>Conformance Report</h1>
 
       <section id="executive-summary">
         <h2>Executive Summary</h2>
+        <p>
+This Implementation Report demonstrates that there are a minimum of two
+independent implementations of each normative statement in the DID Core
+Specification, in accordance with the criteria set forth by the Decentralized
+Identifier Working Group and the World Wide Web Consortium.
```

Comment:
```suggestion
This Implementation Report demonstrates that there are a minimum of two
independent implementations of each normative statement in the DID Core
Specification, in accordance with the criteria and resolutions set forth by 
the Decentralized Identifier Working Group and the World Wide Web 
Consortium.
```

Reply:
Rewording slightly to include the resolutions the WG made around the JSON representation.